include/wlr/types/wlr_foreign_toplevel_management_v1.h

```diff
@@ -84,6 +87,12 @@ struct wlr_foreign_toplevel_handle_v1_activated_event {
 	struct wlr_seat *seat;
 };
 
+struct wlr_foreign_toplevel_handle_v1_fullscreen_event {
+	struct wlr_foreign_toplevel_handle_v1 *toplevel;
+	bool fullscreen;
+	struct wl_resource *output;
```

Comment:
Oh, can we expose the `wlr_output` instead of the `wl_resource`?

Reply:
Oh that is my bad, sorry. When I first wrote this, I forgot that there is a `wlr_output_from_resource`